em-reactor-demo/main.rb

```diff
@@ -1,81 +0,0 @@
-# Very simple, vanilla eventmachine based demo for illustrating reactor
```

Comment:
em-reactor-demo files should not be touched by this PR.

Reply:
Going to reorganize my PR tomorrow to fix this.